stable/postgresql/values.yaml

```diff
@@ -55,6 +55,13 @@ usePasswordFile: false
 #   host all all localhost trust
 #   host mydatabase mysuser 192.168.0.0/24 md5
 
+## Specify a secret containing initdb.d files
+# initdbdSecret:
+
+## Specify initdb.d files mount path
+## Default: /docker-entrypoint-initdb.d
```

Comment:
Is there really a default somewhere?

Reply:
The default is in values.yaml

Reply:
I mean you write (or at least I understand) that there is a default somewhere if no value in values.yaml is provided. Which is wrong. :)
You could just do something like

```yaml
## Specify initdb.d files mount path
## Only used if initdbdSecret is set
# initdbdMountPath: /docker-entrypoint-initdb.d
```

Or even 
```yaml
## Specify initdb.d files mount path
## Only used if initdbdSecret is set
initdbdMountPath: /docker-entrypoint-initdb.d
```
since it's not used if `initdbdSecret` is set.